src/components/Projects/ProjectCard.tsx

```diff
@@ -0,0 +1,37 @@
+import type { Project } from "../../types/type";
+import styles from "../../styles/ProjectCard.module.css";
+
+interface ProjectCardProps {
+  project: Project;
+}

```

Comment:
Nit: since it is a single prop.

```suggestion
interface ProjectCardProp {
  project: Project;
}

```

Reply:
fixed https://github.com/Sakshi-Simform/portfolio/pull/3/commits/4ee0d1e22693a58eb551e6eec848a853179e5b5e